spec.md

```diff
@@ -2490,6 +2490,15 @@ module MyModule :
   out.a <= in ; out.a is of type const UInt
 ```
 
+A sub-field referring to a field whose name is a literal identifier is shown
+below:
+
+``` firrtl
```

Comment:
```suggestion
```firrtl
```

Reply:
Either one of these will work properly and it's used somewhat inconsistently within the document with most of them having a space. The ones just before this use the style with a space, so I'll keep this. 

The reason for this is because this happens to be the setting my editor has when I do a triple-backtick. It prompts me for the format and then adds a space. 🤷‍♀️